rebasehelper/utils.py

```diff
@@ -1198,7 +1199,7 @@ def download_build(cls, client, build_id, destination):
         logs = []
         for _, url in six.iteritems(result.data['results_by_chroot']):
             url = url if url.endswith('/') else url + '/'
-            d = pyquery.PyQuery(url, opener=lambda x: urllib.request.urlopen(x))
+            d = pyquery.PyQuery(url, opener=lambda x: requests.get(x))
```

Comment:
Actually `PyQuery` uses `requests` by default, so removing `opener` argument should be sufficient, see:
https://github.com/gawel/pyquery/blob/master/pyquery/openers.py